### PR TITLE
Updated invisorql to the 2.0 release.

### PR DESCRIPTION
**Pull request by morganestes**
This also adds known dependencies on OS version, architecture, and an additional
package that the quicklook requires to operate.
The `ffmpeg` package is used if installed, but doesn't affect the plugin, so it's
not on the dependencies list.

### CODE REVIEW

**Comment by morganestes** [comment]
Hm, I squashed my commits but wasn't given the open to squash the merge of master. Sorry about that, I know squashing is important to PRs.

File: Casks/invisorql.rb [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
 
-  url 'http://www.pozdeev.com/invisor/InvisorQL.zip'
+  url "http://www.pozdeev.com/invisor/download/InvisorQL-#{version}.dmg"
   name 'Invisor QuickLook Plug-In'
   homepage 'http://www.pozdeev.com/invisor/'
   license :unknown    # todo: change license and remove this comment; ':unknown' is a machine-generated placeholder
+  depends_on :macos => '>= :lion'
+  depends_on :arch => :x86_64
+  depends_on :cask => 'invisor-lite'
+  caveats 'If ffmpeg is installed, invisorql will use it to create thumbnails in QuickLook.
```
Comment (vitorgalvao) [inline Casks/invisorql.rb]: Please remove the caveat. These shouldn’t be used for suggestions.

**Comment by vitorgalvao** [comment]
Are you having trouble squashing?

**Comment by morganestes** [comment]
I removed the caveats line and updated the branch/PR.

I followed the squash guide linked to in the contributing doc and was able to squash my own commits, but was surprised to see the merge to/from master dated Nov. 10, 2014, especially since I opened this branch today. Rebasing didn't show me those commits in the list to pick, or I would've squashed them, too. Any advice on how to keep those merges from showing up in the PRs?

**Comment by vitorgalvao** [comment]
It’s an obtuse process. Doing it a few more times will make it clearer. Merging in the above mentioned pull request. Your contribution is still included (and still credited to you), with the appropriate modifications. Thank you.

**Pull request closed without merging**